aws/sagemaker_studio/rapids_studio_hpo.ipynb

```diff
@@ -0,0 +1,567 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<span style=\"color:#8735fb; font-size:22pt\"> **Demo Overview** </span>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Automated Model Tuning (AMT) also known as Hyper-Parameter Optimization  (HPO) helps to find the best version of a model by exploring the space of possible configurations. While generally desirable, this search is computationally expensive and can feel prohibitive. \n",
+    "\n",
+    "\n",
+    "In the notebook demo below, we show how SageMaker Studio and RAPIDS working together can tackle model tuning by accelerating compute parallelism within a node's GPUs; and simultaneously accelerating the search by leveraging sets of cloud nodes running parallel experiments. \n",
+    "\n",
+    "For more check out our [AWS blog](https://aws.amazon.com/blogs/machine-learning/rapids-and-amazon-sagemaker-scale-up-and-scale-out-to-tackle-ml-challenges/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<span style=\"color:#8735fb; font-size:22pt\"> **Preamble** </span>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get things rolling let's make sure we can query our AWS SageMaker execution role and session as well as our account ID and AWS region."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sagemaker\n",
+    "import string\n",
+    "import random"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "execution_role = sagemaker.get_execution_role()\n",
+    "session = sagemaker.Session()\n",
+    "\n",
+    "account=!(aws sts get-caller-identity --query Account --output text)\n",
+    "region = [session.boto_region_name]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "account, region"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<span style=\"display: block; color:#8735fb; font-size:22pt\"> 1. RAPIDS Cloud ML Container </span>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to use containers with SageMaker, it must reside in Amazon's Elastic Container Registry (ECR) so that AWS SageMaker will be able to leverage the image to build Estimators and run experiments. Normally, in order to use your own container, this requires pulling your Docker container, making any necessary changes, and then publishing it to the ECR. \n",
```

Comment:
Can't we also use containers from the Marketplace?

Reply:
not really with an Estimator object, as the container from Marketplace would also need to contain the training script, etc. in order to look like the container we build for the HPO example. From my understanding, the containers from the Marketplace are more designed to be used as the kernel for a notebook (see comment above). I'll make it more clear by saying something like "In order to use containers with SageMaker Estimators, it must reside in Amazon's ECR"